### DISCUSSION TITLE
I cant seem to do transactions for Accounts on Terra2 that are connected with Keplr (via ledger)

### DISCUSSION THREAD

**Discussion by mohammedpatla** [ping-pub/explorer#302] [Q&A]
I have a Terra2 account on Keplr, that I have over-connected via a ledger (with the cosmos app). I know terra has moved over to the terra ledger app, but I have my assets on the cosmos app, unfortunately. I have imported terra2 address into ping pub but i can't seem to do transactions through the site, it does not allow me to do tx unbond (or bond, or withdraw rewards). I can claim my staking rewards directly using the stake rewards button on keplr directly.

When I click the send button it should send a request to keplr that once i approve on my ledger would send it to the blockchain, but the send button just fails.

Here is an image:
<img width="1297" alt="image" src="https://user-images.githubusercontent.com/16710191/210010261-0ee0b310-d22c-417c-b56b-22f3592e299e.png">

This could be as simple as the endpoint but i am not sure what's breaking this?

**Comment by liangping** [comment]
Any error message shows up when your click send?

**Reply by mohammedpatla** [reply]
Nothing, no error on the app itself it looks like the send is clickable but nothing happens as a response, but looking at the console i get:
```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'height')
    at f.create (app-c3373795.b7b9323b.js:1:9500)
    at v.create (app-c3373795.b7b9323b.js:1:9840)
    at y.create (app-c3373795.b7b9323b.js:1:9994)
    at app-c3373795.b7b9323b.js:1:13209
```

**Comment by liangping** [comment]
seems they didn't update to latest sdk.

**Reply by mohammedpatla** [reply]
oh so do i need to change something?

**Reply by MacD8787** [reply]
all you have to do is open the cosmos app on ledger for these types of  transactuons. when it comes to keplr, using the cosmos app via ledger works on every blockchain especially 2.0
